### PR TITLE
add new sailfish package

### PR DESCRIPTION
**Pull request by bgruening**
Needed for https://github.com/bgruening/galaxytools/pull/254

### CODE REVIEW

File: packages/package_sailfish_0_7_6/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                <actions architecture="x86_64" os="darwin">
+                    <action type="download_by_url">https://github.com/kingsfordgroup/sailfish/releases/download/v0.7.6/SailfishBeta-0.7.6_OSX_10.10.tar.gz</action>
+                    <action type="move_directory_files">
+                        <source_directory>.</source_directory>
+                        <destination_directory>$INSTALL_DIR</destination_directory>
+                    </action>
+                    <action type="set_environment">
+                        <environment_variable action="append_to" name="LD_LIBRARY_PATH">$INSTALL_DIR/lib/</environment_variable>
+                        <environment_variable action="prepend_to" name="PATH">$INSTALL_DIR/bin</environment_variable>
+                    </action>
```
Comment (erasche) [inline packages/package_sailfish_0_7_6/tool_dependencies.xml]: just curious, can these actions to set the environment not be done outside of the actions_group? Just for code deduplication.

File: packages/package_sailfish_0_7_6/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    <action type="shell_command">cd ./build &amp;&amp; make install</action>
+                    <action type="set_environment">
+                        <environment_variable action="append_to" name="LD_LIBRARY_PATH">$INSTALL_DIR/lib/</environment_variable>
+                        <environment_variable action="prepend_to" name="PATH">$INSTALL_DIR/bin</environment_variable>
+                        <environment_variable action="append_to" name="LD_LIBRARY_PATH">$ENV[BOOST_ROOT_DIR]/lib/</environment_variable>
+                    </action>
+                </actions>
+            </actions_group>
+        </install>
+        <readme>Compiling sailfish requires cmake and C++11 compiler (usually gcc).</readme>
```
Comment (erasche) [inline packages/package_sailfish_0_7_6/tool_dependencies.xml]: Please depend on cmake https://github.com/galaxyproject/tools-iuc/tree/master/packages/package_cmake_3_2_3

File: packages/package_sailfish_0_7_6/tool_dependencies.xml (hunk 1 above)
Comment (bgruening) [inline packages/package_sailfish_0_7_6/tool_dependencies.xml]: mh, not sure ...

File: packages/package_sailfish_0_7_6/tool_dependencies.xml (hunk 2 above)
Comment (bgruening) [inline packages/package_sailfish_0_7_6/tool_dependencies.xml]: I updated the help text instead :)

**Comment by bgruening** [comment]
@erasche done!

**Comment by erasche** [comment]
@bgruening looks great!

**Pull request merged**